how-to/integrate-with-sso/integrate-with-auth0/public/manifest.fin.json

```diff
@@ -0,0 +1,45 @@
+{
+	"devtools_port": 9090,
+	"licenseKey": "openfin-demo-license-key",
+	"runtime": {
+		"arguments": "--v=1 --inspect",
+		"version": "28.106.72.17"
+	},
+	"platform": {
+		"uuid": "integrate-with-sso",
+		"icon": "http://localhost:8080/favicon.ico",
+		"autoShow": true,
+		"providerUrl": "http://localhost:8080/platform/provider.html",
+		"preventQuitOnLastWindowClosed": true,
+		"defaultWindowOptions": {}
+	},
+	"snapshot": {
+		"windows": []
+	},
+	"shortcut": {
+		"company": "OpenFin",
+		"description": "A way of showing examples of what OpenFin can do.",
+		"icon": "http://localhost:8080/favicon.ico",
+		"name": "Integrate with SSO - v10.0.0",
+		"target": ["desktop", "start-menu"]
+	},
+	"supportInformation": {
+		"company": "OpenFin",
+		"product": "Workspace Starter - Integrate With SSO - Client",
+		"email": "support@openfin.co",
+		"forwardErrorReports": true
+	},
+	"customSettings": {
+		"auth": {
+			"domain": "openfin-starter.us.auth0.com",
+        "clientId": "cGwCi1YjCKUAqGmmu4jjXffLMtzwUiHj",
```

Comment:
We should not include the info here. It should be:

"domain": "<DOMAIN>". "clientId": "<CLIENTID>", you might have filled it in while testing.

Reply:
Not sure why you are seeing this info. I do not see it in github.

Reply:
You might be looking at the okta version but this is the auth0 manifest.